### PR TITLE
Dfr 1102 common lib

### PR DESCRIPTION
**Pull request by gmikerlemos**
https://tools.hmcts.net/jira/browse/DFR-1102

### CODE REVIEW

**Review by rishikrsharma — APPROVED** [review]
As this is new- over the time - when it comes in action will be able to access the result.

File: src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.time.LocalDate;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+public class AmendedConsentOrder {
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
```
Comment (ruban72) [inline src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java:19]: Could we use a constant here to represent Date format for all date fields?

File: src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/ApplicationNotApproved.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+public class ApplicationNotApproved {
+    private String andAfter;
+    private String othersTextOrders;
+    private List<RefusalReason> reasonForRefusal;
+    private String selectJudge;
+    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd")
```
Comment (ruban72) [inline src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/ApplicationNotApproved.java:24]: Same as above re date format

File: src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java (hunk 1 above)
Comment (gmikerlemos) [reply src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java:19]: I don't know, but I am sure we'll have to change a few things when the actual serialisation is tried.

File: src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/ApplicationNotApproved.java (hunk 2 above)
Comment (gmikerlemos) [reply src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/ApplicationNotApproved.java:24]: answered above

File: src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java (hunk 1 above)
Comment (gmikerlemos) [reply src/main/java/uk/gov/hmcts/reform/finrem/ccd/domain/AmendedConsentOrder.java:19]: Actually we can

**Pull request merged**